app/main.py

```diff
@@ -1,3 +1,113 @@
+from __future__ import annotations
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
```

Comment:
Code quality: You might want to improve the `__repr__` method to include the class name for better debugging output. For example, it could return `Distance(km={self.km})` instead of just `Distance({self.km})`.